### ISSUE TITLE
Implement ability to zoom in x on current trace in scope window

### ISSUE THREAD

**Issue by adamltaylor** [JaneliaSciComp/Wavesurfer#107]
For now, just want it for the current data in the scope window, without the ability to scroll-back.

**Comment by adamltaylor** [comment]
Someone at NIH also mentioned that this would be helpful.

**Comment by adamltaylor** [comment]
N.T. at Brandeis also mentioned that this would be helpful (specificially the y zooming).

**Comment by adamltaylor** [comment]
Nicolas Tritsch at NYU also said that this would be helpful.

**Comment by MattNeuro** [comment]
Is there any update on this? I was just introduced to WaveSurfer by Koen, and this is an absolutely essential feature.

**Comment by adamltaylor** [comment]
Yeah, it's a long-running embarrassment that WaveSurfer lacks this feature, but the prospects for it getting implemented in the near future are not great.  Right now no one is paying for any serious WaveSurfer development.  Sorry.
